integrations/Fullstory/camelcase.js

```diff
@@ -0,0 +1,51 @@
+// util function to convert the input to string type
+function convertToString(input) {
```

Comment:
Please add unit test cases for this module

Reply:
Will add later

Reply:
It'd not be a good idea to use something that is not properly tested.

Reply:
added